src/styles/main.scss

```diff
@@ -1,3 +1,38 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  justify-items: stretch;
+  div {
+    height: 300px;
+  }
+}
+
+@for $i from 1 through 6 {
+  .container > div:nth-child(#{$i})::after {
+    content: "#{$i}";
+    font-size: 100px;
+    font-family: Arial;
+    color: white;
+    background: rgb((255 - 51*($i - 1)), 0, 0);
+    display: flex;
+    height: 100%;
+    align-items: center;
+    justify-content: center;
+  }
+}
+
+@media screen and (max-width: 599px) {
+  .container {
+    grid-template-columns: 1fr;
+  }
+}
+
+@media screen and (max-width: 899px) and (min-width: 600px) {
+  .container {
+    grid-template-columns: 1fr 1fr;
+  }
+}
```

Comment:
try make .css file without code repeat

Reply:
where is repeat? here i make 2 columns

Reply:
not in scss, but excessive repeats in compiled css